src/Lykke.Service.ClientAccountRecovery.Core/Services/IBruteForceDetector.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Lykke.Service.ClientAccountRecovery.Core.Domain;
+
+namespace Lykke.Service.ClientAccountRecovery.Core.Services
+{
+    public interface IBruteForceDetector
```

Comment:
Strange naming :) As for me it is not a brute force protection in any way, just state validation.

Reply:
A name should reflect intentions. The name StateValidator explains nothing.